examples/lessons/10/app.js

```diff
@@ -48,8 +48,9 @@ export default class AppAnimationLoop extends AnimationLoop {
   }
 
   onInitialize({canvas, gl}) {
-    addKeyboardHandler(canvas, currentlyPressedKeys);
-    addMouseHandler(canvas);
+    const eventManager = new EventManager(canvas);
```

Comment:
One line comment explaining why this example went with mjolnir (as opposed to built in event handling?). 

Imagining a user new to luma.gl who has never heard about mjolnir.js (which is not mention ed in the luma docs) reading this code for the first time...